views/scss/qti/_choice.scss

```diff
@@ -210,7 +210,11 @@
         max-height: 100%;
         display: flex;
         justify-content: center;
-        flex-direction: column;
+        justify-content: flex-start;
```

Comment:
I think we need to delete the previous declaration: `justify-content: center;`

Reply:
OK i will change that and push

Reply:
Checking item render (that is used in Preview and works without style issue) found that `div.label-content` contains `div` inside that has `display:block` by default
![image](https://user-images.githubusercontent.com/25976342/118819330-eb3e7200-b8bd-11eb-816e-f45b86fab2fd.png)
In authoring inside `div[data-html-editable=true]` that has `display: flex;`. 
If you put `display:block` it will fix issue

Reply:
so i will try to change the flex to block (in line ) and remove all flex properties